setup.py

```diff
@@ -12,10 +12,17 @@
     author="Yuvi Panda",
     author_email="yuvipanda@gmail.com",
     license="3 Clause BSD",
-    packages=find_packages(exclude="./tests"),
-    python_requires=">=3.6",
-    install_requires=["jupyterhub>=0.8", "oauthlib>=3.1", "escapism>=1.0"],
-    package_data={
-        "": ["*.html"],
-    },  # noqa: E231
+    packages=find_packages(),
+    python_requires=">=3.5",
+    install_requires=[
+        "josepy==1.4.0",
+        "jupyterhub>=1.3.0",
+        "jwcrypto==0.8",
+        "oauthlib>=3.0",
+        "oauthenticator>=0.13.0",
+        "pem==20.1.0",
+        "pycryptodome==3.9.8",
+        "PyJWT==1.7.1",
+        "pyjwkest==1.4.2",
+    ],
```

Comment:
- Lowered the Python requirement from 3.6 to 3.5 - we have perhaps bumped it after you initialized this work?
- What happened to how package_data and package is set? If it was intentional, can you explain a bit about that because I don't understand the past or current choice - of if it should be in one way or another.
- The package pinnings are too tight here I think, use >= instead I'd say
- install_requires escapism was removed, that seems incorrect

I guess in general, why was all those install_requires added now that I think about it? It confuses me that this package would require oauthenticator suddenly - does it?